### PR TITLE
Add twilio text alerts endpoint

### PR DESCRIPTION
**Pull request by AbhiPrasad**
Endpoint for twilio!

We need a credit card/paypal for get the full version, otherwise every text gets a "Sent from your trial twilio account"

### CODE REVIEW

File: controllers/Alerts.js [hunk 1]
```diff
@@ -80,4 +83,27 @@ AlertRouter.put('/edit', function(req, res) {
...
+AlertRouter.post('/text', function(req, res) {
+  const alertToTextId = req.body.id;
+
+  Alert.findById(alertToTextId, function(err, alert) {
+    if (err) return res.status(400).send(err);
+
+    const {name, description} = alert;
+    const date = new Date().getDay();
+
+    User.find({'day': date}, function(err, people) {
```
Comment (AbhiPrasad) [inline controllers/Alerts.js]: `const date = new Date().getDay();` numbers dates from 0-6 from sunday - sat, so it matches perfectly with our current numbering system!

**Comment by theRoughCode** [comment]
@AbhiPrasad We should be able to use our $500 kickstarter credit, but for some reason I can't access my billing page.  Contacting customer support for that. But this works for now, great work!

File: controllers/Alerts.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 const AlertRouter = require('express').Router();
 const mongodb = require('mongodb');
 const Alert = require('../models/Alert');
+const User = require('../models/User');
+require('dotenv').config();
```
Comment (theRoughCode) [inline controllers/Alerts.js:5]: Do we need this here?   It's already being called in `app.js` so not sure if we need to call it twice even though it's in a diff file

File: controllers/Alerts.js [hunk 3]
```diff
@@ -80,4 +83,27 @@ AlertRouter.put('/edit', function(req, res) {
   });
 });
 
+AlertRouter.post('/text', function(req, res) {
+  const alertToTextId = req.body.id;
+
+  Alert.findById(alertToTextId, function(err, alert) {
+    if (err) return res.status(400).send(err);
+
+    const {name, description} = alert;
```
Comment (theRoughCode) [inline controllers/Alerts.js]: Add spacing between braces!

File: controllers/Alerts.js (hunk 1 above)
Comment (theRoughCode) [inline controllers/Alerts.js]: spacing between braces

**Review by theRoughCode — COMMENTED** [review]
Great work, excited to see Twilio come into our app!

left a few comments

File: controllers/Alerts.js (hunk 2 above)
Comment (AbhiPrasad) [reply controllers/Alerts.js:5]: I tried to do it without it, but it was not working so I required it again.

File: controllers/Alerts.js [hunk 4]
```diff
@@ -80,4 +83,27 @@ AlertRouter.put('/edit', function(req, res) {
   });
 });
 
+AlertRouter.post('/text', function(req, res) {
```
Comment (theRoughCode) [inline controllers/Alerts.js:86]: We should also be updating the corresponding alerts object in the database (i.e. sentDate (Amy's PR) and isSent boolean)

**Comment by AbhiPrasad** [comment]
@theRoughCode I completely forgot to remind you. I switched accounts, because I misspelled my blueprint email when signing up (lmao). I'll send you an invite to the account I am currently using. 

We should probably credit the money to this account instead.

File: controllers/Alerts.js (hunk 4 above)
Comment (theRoughCode) [reply controllers/Alerts.js:86]: @AbhiPrasad hold on this.

Comment (theRoughCode) [reply controllers/Alerts.js:86]: Discussing on Slack whats the best approach

Comment (theRoughCode) [reply controllers/Alerts.js:86]: Ok, we've decided to have it done in the admin panel, so no need to set it in this endpoint

**Review by theRoughCode — APPROVED** [review]

**Pull request merged**